testing/jormungandr-testing-utils/resources/explorer/graphql/schema.graphql

```diff
@@ -17,7 +20,7 @@ type Block {
   date: BlockDate!
 
   """The transactions contained in the block"""
-  transactions(first: Int, last: Int, before: IndexCursor, after: IndexCursor): TransactionConnection!
+  transactions(first: Int, last: Int, before: String, after: String): TransactionConnection!
```

Comment:
Is this within conventions on the paginated connection queries?
I guess the custom `IndexCursor` scalar was easy to do with juniper, but if a custom type is not required, leave it like this.

Reply:
Well, to be honest I'm thinking that maybe I should retain the custom scalar, although maybe just as a `String` wrapper, as the semantic here is a bit important.
So, the graphql connection documentation recommends to use opaque cursors, they even recommend base64 encode whatever you use for pagination, so using `String` is actually fine. But when we were designing this originally, we wanted the ability to implement page-based pagination too, and skip to intermediate pages instead of only up and down. So I just named the thing `IndexCursor` and put the comment saying that it was actually an index (or offset) just in case.

Reply:
I'm rather bothered by potential mistakes that were prevented by using newtypes.